tests/ViewTests.elm

```diff
@@ -48,4 +60,14 @@ all =
                     |> TestContext.simulate (Query.find [ Selector.tag "button" ]) Events.Click
                     |> TestContext.expectView
                     |> Query.has [ Selector.tag "p" ]
+        , test "fails when triggering events on a not found element" <|
+            \() ->
+                htmlProgram
+                    |> TestContext.simulate (Query.find [ Selector.tag "foo" ]) Events.Click
+                    |> expectError "Query.find always expects to find 1 element, but it found 0 instead."
+        , test "fails when triggersingevents on an element that does not handle that event" <|
+            \() ->
+                htmlProgram
+                    |> TestContext.simulate (Query.find [ Selector.tag "button" ]) Events.DoubleClick
+                    |> expectError "Failed to decode string"
```

Comment:
"Failed to decode string" seems like a bad error message.  Does a change to this message need to be made in elm-html-test?